<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QDialog" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sawasdee</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Take Your Pills Video Encoder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Take_Your_Pills_Video_Encoder_images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Sawasdee&quot;;</string>
  </property>
  <widget class="QProgressBar" name="progressbarEncode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>557</y>
     <width>220</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar
{
	border: 0px;
	background-image: url(:/images/mainProgressBarEmpty.png);
 }

 QProgressBar::chunk
{
	background-image: url(:/images/mainProgressBarChunk.png);
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupboxToDoList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>215</y>
     <width>320</width>
     <height>170</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
	border: 0px;
	background-color: rgb(16, 16, 16);
}

QCheckBox
{
	color: rgb(240, 240, 240);
}

QCheckBox:disabled
{
	color: rgb(75, 75, 75);
}

QCheckBox:indicator
{
	width: 10px;
	height: 10px;
	background-color: rgb(255, 255, 255);
}

QCheckBox:indicator:checked
{
	background-color: rgb(0, 100, 184);
}

QCheckBox:indicator:unchecked:disabled
{
	background-color: rgb(100, 100, 100);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkboxPreview">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>16</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Preview</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox360">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Standard Definition (640x360)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox720">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>76</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> High Definition (1280x720)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox1080">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>106</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> High Definition (1920x1080)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxUserDefined">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>136</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> User-defined video</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupboxSetupPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>320</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
	border: 0px;
	background-color: rgb(16, 16, 16);
}

QLabel
{
	color: rgb(240, 240, 240);
}

QLabel:disabled
{
	color: rgb(75, 75, 75);
}

QToolButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/mainSmallButtonNormal.png);
}

QToolButton:hover
{
	background-image: url(:/images/mainSmallButtonHover.png);
}

QToolButton:pressed
{
	background-image: url(:/images/mainSmallButtonPressed.png);
}

QToolButton:disabled
{
	color: rgb(75, 75, 75);
	background-image: url(:/images/mainSmallButtonPressed.png);
}

QDoubleSpinBox
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:disabled
{
	background-color: rgb(100, 100, 100);
}

QDoubleSpinBox:up-button
{
	image: url(:/images/mainSpinBoxUpButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QDoubleSpinBox:up-button:pressed 
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:up-button:disabled
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(100, 100, 100);
}

QDoubleSpinBox:down-button
{
	image: url(:/images/mainSpinBoxDownButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QDoubleSpinBox:down-button:pressed 
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:down-button:disabled
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(100, 100, 100);
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelPreviewAt">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>14</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Preview at</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doublespinboxUserPreviewTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>11</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeconds">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>14</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbuttonUserPreviewTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>11</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> ...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupboxSetupUserDefined">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>457</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
	border: 0px;
	background-color: rgb(16, 16, 16);
}

QLabel
{
	color: rgb(240, 240, 240);
}

QLabel:disabled
{
	color: rgb(75, 75, 75);
}

QDoubleSpinBox
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:disabled
{
	background-color: rgb(100, 100, 100);
}

QSpinBox
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QSpinBox:disabled
{
	background-color: rgb(100, 100, 100);
}

QSpinBox:up-button
{
	image: url(:/images/mainSpinBoxUpButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QSpinBox:up-button:pressed 
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QSpinBox:up-button:disabled
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(100, 100, 100);
}

QSpinBox:down-button
{
	image: url(:/images/mainSpinBoxDownButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QSpinBox:down-button:pressed 
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QSpinBox:down-button:disabled
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(100, 100, 100);
}

QSpinBox
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QSpinBox:disabled
{
	background-color: rgb(100, 100, 100);
}

QDoubleSpinBox
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:disabled
{
	background-color: rgb(100, 100, 100);
}

QDoubleSpinBox:up-button
{
	image: url(:/images/mainSpinBoxUpButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QDoubleSpinBox:up-button:pressed 
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:up-button:disabled
{
	image: url(:/images/mainSpinBoxUpButtonBlack.png);
	background-color: rgb(100, 100, 100);
}

QDoubleSpinBox:down-button
{
	image: url(:/images/mainSpinBoxDownButtonWhite.png);
	background-color: rgb(0, 100, 184);
}

QDoubleSpinBox:down-button:pressed 
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(255, 255, 255);
}

QDoubleSpinBox:down-button:disabled
{
	image: url(:/images/mainSpinBoxDownButtonBlack.png);
	background-color: rgb(100, 100, 100);
}

</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelResizeTo">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>15</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Resize to</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>15</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeepOnly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Keep only</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPoints">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>49</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doublespinboxUserCutTimeStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>46</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doublespinboxUserCutTimeStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>46</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
    <property name="value">
     <double>99999.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinboxUserResizeResolutionX">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>11</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinboxUserResizeResolutionY">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>11</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushbuttonEncode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>557</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/mainEncodeButtonNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/mainEncodeButtonHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/mainEncodeButtonPressed.png);
}

QPushButton:disabled
{
	color: rgb(75, 75, 75);
	background-image: url(:/images/mainEncodeButtonPressed.png);
}</string>
   </property>
   <property name="text">
    <string>    Encode</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>127</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Take_Your_Pills_Video_Encoder_images.qrc">:/images/mainLogo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Take_Your_Pills_Video_Encoder_images.qrc">:/images/mainBackground.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupboxInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
	border: 0px;
	background-color: rgb(16, 16, 16);
}

QLineEdit
{
	border: 0px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QLineEdit:disabled
{
	background-color: rgb(100, 100, 100);
}

QToolButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/mainSmallButtonNormal.png);
}

QToolButton:hover
{
	background-image: url(:/images/mainSmallButtonHover.png);
}

QToolButton:pressed
{
	background-image: url(:/images/mainSmallButtonPressed.png);
}

QToolButton:disabled
{
	color: rgb(75, 75, 75);
	background-image: url(:/images/mainSmallButtonPressed.png);
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineeditUserFileInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>266</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string> Select a video file</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbuttonUserFileInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>11</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineeditBasename">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>46</y>
      <width>266</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Enter a basename</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbuttonBasename">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>46</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> ?</string>
    </property>
   </widget>
  </widget>
  <zorder>labelBackground</zorder>
  <zorder>progressbarEncode</zorder>
  <zorder>groupboxToDoList</zorder>
  <zorder>groupboxSetupPreview</zorder>
  <zorder>groupboxSetupUserDefined</zorder>
  <zorder>pushbuttonEncode</zorder>
  <zorder>labelLogo</zorder>
  <zorder>groupboxInput</zorder>
 </widget>
 <resources>
  <include location="Take_Your_Pills_Video_Encoder_images.qrc"/>
 </resources>
 <connections/>
</ui>
